docs/examples/quick-start-with-terraform.md

```diff
@@ -0,0 +1,212 @@
+# Using Terraform to play with Liqo
+
+Terraform is a widely used Infrastructure as Code (IaC) tool that allows engineers to define their software infrastructure in code.\
+This tutorial aims at presenting how to set up an environment with Liqo installed via Terraform and explore the Liqo capabilities through its Provider and Resources.
+
+## Provision the infrastructure
+
+Before starting the tutorials below, you should ensure the following software is installed on your system:
+
+- [**Docker**](https://docker.io), the container runtime.
+- [**Kubectl**](https://kubernetes.io/docs/tasks/tools/install-kubectl/), the command-line tool for Kubernetes.
+- [**Terraform**](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli), the command-line tool for Terraform.
+
+Then, let's open a terminal on your machine and launch the following script, which creates the infrastructure used in this example (i.e., two KinD clusters, peered with Liqo), with all playground already set up.\
+This tutorial will present a detailed description about how this result is achieved, by analyzing the most notable parts of the Terraview infrastructure definition file that refer to Liqo.
+
+{{ env.config.html_context.generate_clone_example_tf('quick-start-terraform') }}
+
+## Analyze the infrastructure and code
+
+The previous command created two KinD clusters, peered with each other. Furthermore, you offloaded a namespace in Liqo virtual cluster.\
+In more details the offloaded namespace created in local cluster will leverage also on virtual one which isa resource abstraction of remote cluster.
+KinD was chosen because of there are no configuration requirements such as account creation or resource costs, therefore all the presented functionalities works also on other clusters, e.g., the ones operated by public cloud providers.
```

Comment:
```suggestion
This example is provisioned on KinD, since it requires no particular configurations (e.g., concerning accounts), and does not lead to resource costs. 
Yet, all the presented functionalities work also on other clusters, e.g., the ones operated by public cloud providers.
```